plugin/restore/mysql/mysql.go

```diff
@@ -349,6 +349,8 @@ func (r *Restore) SwapPITRDatabase(ctx context.Context, database string, suffixT
 		return pitrDatabaseName, pitrOldDatabase, err
 	}
 
+	defer db.Close()
+
```

Comment:
```suggestion
```

The connection is closed here
https://github.com/bytebase/bytebase/blob/dca0de4f2edebf696a74d988fc2b67cfd87a3305/server/task_executor_pitr_cutover.go#L44